.dockerignore

```
@@ -1 +1,4 @@
-.env
\ No newline at end of file
+.env
+
+configs/secrets*
+!configs/secrets.example*
```

let's be explicit about secrets files

```suggestion
!configs/secrets.example
```

---

we can actually ignore everything in the docker container, it doesn't need the example file

---

changed